routes/images.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express')
+const router = express.Router()
+const debug = require('debug')('k8s-autodeploy:services')
+const {executeCommand, deployment} = require('../util/commands')
+const {commonErrorHandler} = require('../util/errors');
+
+// Restart all deployments that run a specific image
+router.post('/:images/rollout', async function (request, response, next) {
+  const dockerTag = request.body.push_data.tag
+  const images = request.params.images.split(',')
+
+  debug('NEW rollout request. Images => %s, DockerTag => %s', request.params.images, dockerTag)
+  // Gets all deployments running any of the given images separated by space
+  const command = `kubectl get deployments -o jsonpath="{range .items[*]}{.metadata.name}{' '}{.spec.template.spec.containers[*].image}{'\\n'}{end}" | grep ${images.join("\|")} | awk '{print $1}' ORS=' '`
```

Comment:
I believe for this to work with multiple images you need to wrap the regex in quotes. Otherwise it will look for a line that contains the literal `service1\|service2` instead of `either "service1" or "service2"`.
```suggestion
  const command = `kubectl get deployments -o jsonpath="{range .items[*]}{.metadata.name}{' '}{.spec.template.spec.containers[*].image}{'\\n'}{end}" | grep "${images.join("\|")}" | awk '{print $1}' ORS=' '`
```